opus_codec_registration.src.html

```diff
@@ -115,6 +121,8 @@
 <xmp>
 dictionary OpusEncoderConfig {
   OpusBitstreamFormat format = "opus";
+  OpusSignal signal = "auto";
+  OpusApplication application = "audio";
```

Comment:
There is no default in libopus for `OPUS_SET_APPLICATION` (unlike the default of `"auto"` for `OPUS_SET_SIGNAL`).  I think `"audio"` is the most generic/least surprising as a default.